samples/net/sockets/echo_client/sample.yaml

```diff
@@ -0,0 +1,10 @@
+common:
+  harness: net
+  tags: net
+sample:
+  description: Test network sockets using a client/server
+    sample
+  name: Socket Echo Client
+tests:
+  test:
+    platform_whitelist: qemu_x86
```

Comment:
I think we do not need to whitelist qemu_x86, the sample app should run in all the platforms that support networking.
For automatic selection of suitable platforms you can add `depends_on: netif` to common section.

Reply:
A newbie question: in case I remove `whitelist` entry, `tests` section will be empty:
```
common:
  harness: net
  tags: net
  depends_on: netif
sample:
  description: Test network sockets using a client/server
    sample
  name: Socket Echo Server
tests:
  test:
```
What should I put in there? Or is it ok for it to be empty, or should I remove it at all?

Reply:
> What should I put in there? Or is it ok for it to be empty, or should I remove it at all?

I am not really familiar with various syntax details in these yaml files. If there is only one test section, then you could put some of the common tags there. For example "tags: net" could be placed there, if it cannot be left empty.

Reply:
Ok, I'll do something like this, let's see what shippable with that...